hammr/commands/quota/quota.py

```diff
@@ -62,7 +62,11 @@ def do_list(self, args):
                     if quota.limit != -1:
```

Comment:
Limit value is not checked, it may cause runtime exceptions if there is a division by zero

Reply:
On the next patch, I'll check if quota.limit is <= than 0 to avoid a division by zero.